intelmq/bots/parsers/abusech/parser_ransomware.py

```diff
@@ -40,7 +40,15 @@ def process(self):
                         ev.add('classification.type', 'c&c')
                         ev.add('time.source', nrow[0] + ' UTC', overwrite=True)
                         ev.add('status', nrow[5])
-                        ev.add('source.ip', nrow[7])
+                        # in the feed sometimes a ',' is seen before srcip
+                        # perhaps it is an error on feeds part
+                        # however we have to deal with it
+                        src_ip = nrow[7]
+                        if src_ip and len(src_ip) > 8 and src_ip[0] == ',':
```

Comment:
This takes care of an additional case where the feed can contain values like `,104.` as IP address. Generally these are fix in the next fetch but in case it comes, the current processing will not stop. This is not the best way to take care of it though.